tftrt/examples/image-classification/image_classification.py

```diff
@@ -63,13 +63,16 @@ def __init__(self, target_duration=None, iteration_limit=None):
     def before_run(self, run_context):
         if not self.start_time:
             self.start_time = time.time()
-            print("    running for target duration from %d", self.start_time)
+            if self.target_duration:
+                print("    running for target duration {} seconds".format(self.target_duration))
+                print(" from {}".format(time.asctime(time.localtime(self.start_time))))
```

Comment:
I think it's better to print these two on the same line.

Reply:
you are correct. I meant to split only the code over two lines, not the message.